deploy/000-deploy-contracts.js

```diff
@@ -34,7 +34,16 @@ const {
 
 const func = async ({ getNamedAccounts, deployments, getChainId, ethers }) => {
   const chainId = await getChainId()
-  const gasStation = await initializeGasStation(ethers.provider)
+
+  const gasStation = await initializeGasStation(ethers.provider).catch(() => {
+    console.warn('Falling back to dummy gas station.')
+    return {
+      async getCurrentFees() {
```

Comment:
I dont see that function defined anywhere

Reply:
This function is defined in @solid-world/gas-station package. Also notice that this is creation of an object with `getCurrentFees` method, not an execution of `getCurrentFees` function.